clinica/iotools/bids_utils.py

```diff
@@ -467,15 +467,20 @@ def _write_bids_dataset_description(
         bids_desc.write(to=f)
 
 
-def _write_readme(bids_dir: Union[str, Path]) -> None:
-    """Write `README`file at the root of the BIDS directory."""
-    import clinica
+def _write_readme(
+    study_name: str,
+    data_dict: dict,
+    bids_dir: Union[str, Path],
+    bids_version: Optional[str] = None,
```

Comment:
This is not used and should be removed.

Reply:
Unless we want to specify the minimum BIDS version of the dataset in the readme maybe?

Reply:
It is called by the dataset description though

Reply:
`bids_version` is not used in this function